docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -222,18 +223,14 @@ Zowe CLI uses a "command option order of precedence" that lets your service defi
 The CLI checks for option values in the following order. If not found, the next location is checked:
 1. Options you define explicitly on the command-line
 2. Environment variables
-3. Service array definitions
-4. Base array definitions
+3. Service type profiles
+4. Base type profiles
 5. If no value is found, the default value for the option is used.
 
 If you have two services that share the same username and password on the mainframe, you can define your username and password just once in the base array, leaving those fields blank in each service definition.
```

Comment:
In this sentence and the next I suggest that we replace the references to 'supplying a blank or empty user name or password', with a statement that 'the user name and password fields are not supplied' in each service definition.